libfuse/start.go

```diff
@@ -85,19 +86,29 @@ func startMounting(ctx context.Context,
 // Start the filesystem
 func Start(options StartOptions, kbCtx libkbfs.Context) *libfs.Error {
 	// Hook simplefs implementation in.
-	options.KbfsParams.CreateSimpleFSInstance = simplefs.NewSimpleFS
+	createSimpleFS := func(
+		libkbfsCtx libkbfs.Context, cfg libkbfs.Config) (rpc.Protocol, error) {
+		return keybase1.SimpleFSProtocol(
+			simplefs.NewSimpleFS(libkbfsCtx.GetGlobalContext(), cfg)), nil
+	}
 	// Hook git implementation in.
 	shutdownGit := func() {}
-	options.KbfsParams.CreateGitHandlerInstance =
-		func(config libkbfs.Config) (i keybase1.KBFSGitInterface) {
-			i, shutdownGit = libgit.NewRPCHandlerWithCtx(
-				kbCtx, config, &options.KbfsParams)
-			return i
-		}
+	createGitHandler := func(
+		libkbfsCtx libkbfs.Context, config libkbfs.Config) (rpc.Protocol, error) {
```

Comment:
above you use `cfg` and here you use `config`. I don't have a preference as to which one, but they should be consistent.